core/build.gradle

```diff
@@ -38,7 +38,7 @@ tasks.withType(Jar).named(kotlin.jvm().artifactsTaskName) {
     manifest {
         attributes(
                 "Implementation-Version": version,
-                "Require-Kotlin-Version": "1.4-RC",
+                "Require-Kotlin-Version": "1.4.30-M1",
```

Comment:
NB: bug fixes do not require raising minimal kotlin version, but for this case, we wanted to do it anyway